changelog.md

```diff
@@ -96,6 +96,7 @@
 * Added scripts for the new docker image that facilitate the start up automation.
 * Adjusted the docker pipeline slightly to pull the version on PR from the app.go file.
 * [1781](https://github.com/zeta-chain/node/pull/1781) - add codecov coverage report in CI
+* Fix docker repo typo.
```

Comment:
```suggestion
```
I don't think we need a changelog edit for this. Also we should be more specific in the changelog edit and link the PR